app/jobs/sample_job.rb

```
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class SampleJob < ApplicationJob
+  queue_as :default
+
+  def perform(*args)
+    puts "I'm starting job"
```

rubocop (1.68.0)

[convention] Do not write to stdout. Use Rails's logger if you want to log.

(Rails/Output)